binding.c

```diff
@@ -514,21 +514,21 @@ NAPI_METHOD(udx_napi_network_interfaces) {
   napi_create_array(env, &result);
 
   while (i < count) {
-    uv_interface_address_t interface = interfaces[i++];
```

Comment:
I don't even know what to think... https://stackoverflow.com/a/25234279/12524929

Reply:
ah yes - naming paranoia. hilarious